src/views/payment-sheet-views/base-paypal-view.js

```diff
@@ -104,6 +104,8 @@ BasePayPalView.prototype.initialize = function () {
 
 BasePayPalView.prototype.requestPaymentMethod = function () {
   this.model.reportError('paypalButtonMustBeUsed');
+
+  return Promise.reject(new DropinError(constants.errors.NO_PAYMENT_METHOD_ERROR));
```

Comment:
```suggestion
  return BaseView.prototype.requestPaymentMethod.call(this)
```

Reply:
Changing it to call the method in the BaseView should amount to the same thing, but I think it's better to defer to the default behavior here (in case that default behavior for requesting a payment method on a non-requestable view changes).

Reply:
Good idea, will make that change.